coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/copy/ProposalDevelopmentCopyController.java

```diff
@@ -46,6 +46,13 @@ public ModelAndView copy(@ModelAttribute("KualiForm") ProposalDevelopmentDocumen
         return returnToDocument(form, newDoc.getDocumentNumber());
     }
 
+
+    @Transactional @RequestMapping(value = "/proposalDevelopment", params = "methodToCall=displayCopyDialog")
+    public ModelAndView displayCopyDialog(@ModelAttribute("KualiForm") ProposalDevelopmentDocumentForm form) throws Exception {
+        ((ProposalDevelopmentViewHelperServiceImpl)form.getView().getViewHelperService()).populateQuestionnaires(form);
+        return getModelAndViewService().showDialog(COPY_DIALOG,true,form);
```

Comment:
Thats annoying. In the future we should find a way to more easily bind questionnaires to the object graph so we don't have to do this kind of thing everything.